installer/conf/omsagent.d/npmd.conf

```diff
@@ -0,0 +1,14 @@
+<source>
+    type npmd
+</source>
+
+<match npmd.oms.blob>
```

Comment:
you don't need a special output plugin in the configuration. it can be processed by the one in omsagent.conf